readme.txt

```diff
@@ -20,6 +20,7 @@ The `[ucf-section]` shortcode has several options:
 * `class` - CSS class(es) to assign to the generated `<section>` element. By default, sections will always have the `.ucf-section-SLUG` class assigned to them.
 * `title` - a descriptive title to use both as an accessible `aria-label` value for screenreaders in the generated `<section>` element, and in navbar links generated by the [Automatic Sections Menu Shortcode](https://github.com/UCF/Section-Menus-Shortcode).
 * `section_id` - a value to assign as the `ID` attribute of the generated `<section>` element.
+* `random_from_tag` - if provided a tag, a random section will be chosen to display from sections that are assigned the given tag.
```

Comment:
Let's make sure to specify here that a tag slug is required.

Reply:
Good idea, done, thanks!